src/index.ts

```diff
@@ -234,6 +232,8 @@ function updateActiveOrientationFromLocalStorage(): void {
 		utility.freezeOverlays();
 		utility.continueOverlays();
 	}
+	
+	console.log(gauges)
```

Comment:
I assume this log was used for debugging purposes but it should get removed as it will only create clutter when asking people for screenshots of their console when troubleshooting.

Reply:
Correct, that was how I found that infinite recursion. xd Will remove.